### PR TITLE
Update unittests

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates the unittests to add more coverage, as well as properly ignore blocks of code that are too challenging/impossible/not worthwhile to test.

### CODE REVIEW

**Comment by foolip** [comment]
@queengooborg sorry I neglected this. Can you rebase?

**Comment by queengooborg** [comment]
Actually, can you help with that?  I can't modify v1 package locks now due to a [bug with NPM v6](https://stackoverflow.com/questions/69452504/zlib-error-when-attempting-to-run-npm-install-or-yarn)...

File: unittest/app/app.js [hunk 1]
```diff
@@ -249,6 +249,11 @@ describe('/tests/', () => {
     assert.equal(res.status, 200);
   });
 
+  it('get all tests, ignore CSS', async () => {
+    const res = await agent.get('/tests/?ignore=css');
+    assert.equal(res.status, 200);
```
Comment (foolip) [inline unittest/app/app.js:254]: Should something be asserted here that's different from the previous step? As it is, this test would pass even if the `ignore` parameter is unused.

File: unittest/unit/build.js [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 'use strict';
 
-import chai, {assert, expect} from 'chai';
+import chai, {assert} from 'chai';
```
Comment (foolip) [inline unittest/unit/build.js:17]: Yay, I like `assert` better than `expect` :)

File: unittest/unit/build.js [hunk 3]
```diff
@@ -1302,28 +1357,39 @@ describe('build', () => {
...
+          properties: {
+            grid: {}
+          }
         }
-      }
-    };
+      };
 
-    const customCSS = {
+      const customCSS = {
```
Comment (foolip) [inline unittest/unit/build.js:1392]: Accidental indentation here? Doesn't this fail some lint rule?

**Review by foolip — APPROVED** [review]

File: unittest/app/app.js (hunk 1 above)
Comment (queengooborg) [reply unittest/app/app.js:254]: We should probably also check to make sure we're loading the correct tests as well in the previous two steps.  I'm not sure how to do it at this time though, so I've added some TODO comments to remind us later.

**Pull request merged**